source/views/components/tabbed-view/tabbed-view.ios.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import React from 'react'
+import {TabBarIOS} from 'react-native'
+import Icon from 'react-native-vector-icons/Ionicons'
+import {tracker} from '../../../analytics'
+import {styles} from './styles'
+import type {TabbedViewPropsType, TabPropsType} from './types'
+import * as c from '../../components/colors'
+
+export class TabbedView extends React.Component {
+  state: {selectedTab: string};
+
+  componentWillMount() {
+    this.onChangeTab(this.props.tabs.filter(Boolean)[0].id)
```

Comment:
I cannot fault you for this actually working, and I understand that it gives you the option of nullable tabs...